guvnor-project/guvnor-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/PluginUpdater.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2015 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.guvnor.common.services.project.backend.server;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.maven.model.Plugin;
+
+public class PluginUpdater {
+
+    private final List<Plugin> plugins;
+
+    public PluginUpdater( List<Plugin> plugins ) {
+        this.plugins = plugins;
+    }
+
+    public List<Plugin> update( ArrayList<org.guvnor.common.services.project.model.Plugin> from ) {
+
```

Comment:
Now that I took a look. The method above should be here. Adding test coverage and a fix.